x-pack/test/security_solution_endpoint/page_objects/endpoint_page.ts

```diff
@@ -17,9 +17,11 @@ export function EndpointPageProvider({ getService, getPageObjects }: FtrProvider
      * Navigate to the Endpoints list page
      */
     async navigateToEndpointList(searchParams?: string) {
-      await pageObjects.common.navigateToApp('securitySolution', {
-        hash: `/management/endpoints${searchParams ? `?${searchParams}` : ''}`,
-      });
+      await pageObjects.common.navigateToUrl(
```

Comment:
We probalbly should switch to using `pageObjects.common.navigateToUrlWithBrowserHistory()` instead (we were using it before under the `endpoint` plugin)